installing.rst

```diff
@@ -293,42 +298,59 @@ If you have Gauge and your language runner installed, then see how you can :ref:
 Uninstallation
 --------------
 
-Gauge
-^^^^^^^^
-
 .. warning::
    If you plan to remove Gauge and the installed plugins, follow the steps in :ref:`plugins-uninstallation` first.
 
-
+Gauge
+^^^^^
 To uninstall Gauge, run the following commands:
 
-OS X/Linux
-++++++++++
+Windows
++++++++
+Uninstallation using `choco <https://github.com/chocolatey/choco/wiki/CommandsUninstall>`__
 
 .. code-block:: console
 
-   rm -rf /usr/local/bin/gauge /usr/local/bin/gauge_screenshot ~/.gauge
+    choco uninstall gauge
 
-If Gauge is installed in custom location, user will have to remove
-corresponding files/directory.
+OS X
+++++
+Uninstallation using `brew <https://docs.brew.sh/FAQ.html#how-do-i-uninstall-a-formula>`__
 
-Windows
-++++++++++
+.. code-block:: console
+
+    brew uninstall gauge
+
+Debian, Ubuntu
+++++++++++++++
+Uninstallation using `apt-get <https://linux.die.net/man/8/apt-get>`__
+
+.. code-block:: console
+
+    sudo apt-get remove gauge
+
+CentOS/RHEL
++++++++++++
+Uninstallation using `yum <https://www.centos.org/docs/5/html/5.1/Deployment_Guide/s1-yum-useful-commands.html>`__
+
+.. code-block:: console
+
+    yum remove gauge
 
-Run the executable ``uninst.exe`` found in Gauge install location.
+Fedora
+++++++
+
+.. code-block:: console
+
+    dnf remove gauge
+
+If Gauge is installed in custom location, user will have to remove
```

Comment:
we should be consistent with the grammar, the docs mix reference in 1st and 3rd person.